recipes/zppy-interfaces/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "zppy-interfaces" %}
+{% set version = "0.1.1" %}
+{% set python_min = "3.9" %}
```

Comment:
```suggestion
```

Reply:
Oops, I needed that for local testing and forgot to remove it.  Thanks!